app/assets/stylesheets/responsive/_request_layout.scss

```diff
@@ -318,6 +318,19 @@ input.cplink__field {
   line-height: 1em;
 }
 
+a.cplink__button {
+  padding: 0.5em;
+  &:hover,
+  &:active,
+  &:focus {
+    border-left: 0;
```

Comment:
that doesn't look quite right as it causes the border to appear cropped **but** the only alternative I could come up with caused the button to move sideways on rollover